pkg/providers/amifamily/ami.go

```diff
@@ -79,25 +81,25 @@ func (a AMIs) Sort() {
 	})
 }
 
-func (a AMIs) String() string {
+func String(amis []v1beta1.AMI) string {
```

Comment:
Can we rename this to something like PrettyFormat() or something else? Calling it `String()` could make a user think that the intention was to make this a member function.